app/models/katello/event_queue.rb

```diff
@@ -0,0 +1,25 @@
+module Katello
+  class EventQueue < Katello::Model
+    self.table_name = 'katello_event_queue'
+
+    @repository_types = {}
```

Comment:
apparently my brain checked out when i was creating this class :)  That should be event_types